.github/workflows/run-test.yaml

```diff
@@ -112,20 +128,14 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: ${{ matrix.architecture }}
 
-      - name: Remove greenlet
-        if: ${{ matrix.no-greenlet == 'true' }}
-        shell: pwsh
-        run: |
-          (cat setup.cfg) | %{$_ -replace "^\s*greenlet.+",""} | set-content setup.cfg
```

Comment:
was this removed because it's not needed anymore?    The reason we have to be careful with greenlet is because as new python alphas and betas come out, greenlet fails to build until they get it working.   I think we should have this covered in our tox.ini file and that greenlet doesnt install by default.   So this is in theory fine, but im also not good at reading github workflow yamls

Reply:
This assumes that greenlet is listed in `setup.cfg` which is not the case anymore so it's probably old stuff that remained after migrating to pyproject.toml?

The current `tox` config should be able to handle no-greenlet. Though it's not used on CI at all.

Reply:
greenlet is a dependency of the `[asyncio]` optional target in pyproject.toml .  oh right so that rule wasn't doing anything anyway, it was grepping out setup.cfg.  so yeah